src/pages/ConnectWallet/ConnectWallet.tsx

```diff
@@ -283,7 +241,7 @@ export const ConnectWallet = () => {
           zIndex={1}
           width='full'
           maxWidth='400px'
-          bgImage={NightSky}
+          background='linear-gradient(135deg, #0052FF 0%, #0052FF 40%, #00D2FF 100%)'
```

Comment:
![Frame 12](https://github.com/user-attachments/assets/0100c383-758c-4d10-9e6f-fc98642319c4)

I think you can use this SVG instead of a linear-gradient background if you wan't to follow the mockups

Reply:
I got a JPG from Beard, which I used in https://github.com/shapeshift/web/pull/7793/commits/f6c0fcd3dee9a0a39ea57eed30a38fc8211ec892